index.d.ts

```diff
@@ -19,6 +29,117 @@ export interface MonacoTailwindcssOptions {
    * worker.
    */
   tailwindConfig?: TailwindConfig | string
+  /**
+   * Extend Intellisense's settings.
+   * 
+   * Default values are based on the VS Code extension.
+   * 
+   * ``` typescript
+   * const defaultSettings = {
+      editor: {tabSize: 2},
+      tailwindCSS: {
+        emmetCompletions: false,
+        classAttributes: ["class", "className", "ngClass"],
+        codeActions: true,
+        hovers: true,
+        suggestions: true,
+        validate: true,
+        colorDecorators: true,
+        rootFontSize: 16,
+        showPixelEquivalents: true,
+        includeLanguages: {},
+        files: {exclude: []},
+        experimental: {
+          classRegex: [],
+          configFile: {}
+        },
+        lint: {
+          cssConflict: "warning",
+          invalidApply: "error",
+          invalidScreen: "error",
+          invalidVariant: "error",
+          invalidConfigPath: "error",
+          invalidTailwindDirective: "error",
+          recommendedVariantOrder: "warning"
+        },
+      }
+    }
+   */
+  intellisense?: Partial<{
```

Comment:
VSCode calls this settings. LSP calls it configuration. Since we already have so many things called `configuration`, I’m leaning towards calling this property `settings`.

Reply:
Sounds good